requests_oauth2client/dpop.py

```diff
@@ -199,12 +273,26 @@ def dpop_jkt(self) -> str:
     def proof(self, htm: str, htu: str, ath: str | None = None, nonce: str | None = None) -> jwskate.SignedJwt:
         """Generate a DPoP proof.
 
+        Proof will contain the following claims:
+
+            - The HTTP method (`htm`), target URI (`htu`), and Access Token hash (`ath`) that are passed as parameter.
+            - The `iat` claim will be generated by the configured `iat_generator` (default to current datetime).
+            - The `jti` claim will be generated by the configured `jti_generator`, which defaults to a random UUID4.
```

Comment:
Either "(defaults to [...])" or ", which defaults to […]" is fine, but be consistent.